i18n/en.i18n.json

```diff
@@ -274,6 +274,8 @@
   "Login_with" : "Login with %s",
   "login_with" : "Or login directly with",
   "Logout" : "Logout",
+  "Logout_Others" : "Logout Others",
```

Comment:
Ok, I will change it. But then, the button might occupy more space to accommodate the text. Hope that's okay.